Library/Homebrew/cask/url.rb

```diff
@@ -1,15 +1,31 @@
-# typed: true # rubocop:todo Sorbet/StrictSigil
+# typed: strict
 # frozen_string_literal: true
 
 require "source_location"
 require "utils/curl"
 
 module Cask
+  BlockReturn = T.type_alias do
+    T.any(URI::Generic, String, [T.any(URI::Generic, String), T::Hash[Symbol, T.untyped]])
+  end
```

Comment:
I don't think this should be this high up the namespace chain. I'd say preferrably under `BlockDSL` but certainly no higher than `URL`.

Reply:
Oh, absolutely, copy pasta fail